docs/word/citation-management.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Sample: Manage citations through your Word add-in'
+description: Learn how to create an add-in that manages citations in a Word document.
+ms.date: 11/15/2023
+ms.topic: sample
+ms.localizationpriority: medium
+---
+
+# Sample: Manage citations in a Word document using your Word add-in
+
+Citation management is an important aspect of documents, particularly in academia and education. Each citation style has its own guidelines for how citations should be marked in a document as well as where and how the sources should be noted. Examples of citation styles are [APA](http://apastyle.apa.org/) and [MLA](https://www.mla.org/MLA-Style).
+
+This article shows how to create an add-in that showcases the basics of managing citations in a Word document. The add-in displays the references loaded from the user's .bib file then the user can insert selected references into the document.
```

Comment:
A couple suggestions to make this more concise. I'm also not sure we're showing people "how" to do something, as much as showing them a working version. Maybe that nuance isn't important, but something to consider.
```suggestion
This article features a sample add-in that manages citations in a Word document. The add-in displays the references loaded from the user's .bib file that the user selects to cite in their document.
```